packages/core/src/browser/keyboard/browser-keyboard-layout-provider.ts

```diff
@@ -73,11 +73,12 @@ export class BrowserKeyboardLayoutProvider implements KeyboardLayoutProvider, Ke
         for (const [code, key] of layoutMap.entries()) {
             tester.updateScores({ code, key });
         }
-        const result = tester.getTopScoringCandidates();
-        if (result.length > 0) {
-            return result[0];
+        const topScoring = tester.getTopScoringCandidates();
+        const result = topScoring.find(l => l !== this.macUS && l !== this.winUS);
+        if (result) {
```

Comment:
`result` is `undefined`

<img width="396" alt="Screen Shot 2019-04-25 at 09 57 45" src="https://user-images.githubusercontent.com/914497/56719252-a8425c80-6740-11e9-9543-11109bbb7caf.png">

Reply:
Yes, that's intended. In that case it should fall back to `this.createDefaultLayout()`.

Reply:
@spoenemann, can you check the predicate of `find` it looks bogus.

Reply:
The problem is, there are different US layouts :-D

You seem to have the `` ` `` key between `shift` and `Z`, but on the static layout I created it's left of `1` (where you have `§`).

Reply:
With my change an empty mapping is used, so no key is remapped.

If that does still not work, you need to create a static layout with your keyboard.

Reply:
@spoenemann, you're right! I'll create such a static layout.

this is from the MBP keeb:
<img width="1285" alt="Screen Shot 2019-04-25 at 10 34 38" src="https://user-images.githubusercontent.com/914497/56721887-22291480-6746-11e9-870a-a738d2b9426f.png">

this is from my external keeb:
<img width="1291" alt="Screen Shot 2019-04-25 at 10 33 30" src="https://user-images.githubusercontent.com/914497/56721888-22291480-6746-11e9-9a78-c1972e5e1850.png">